### PR TITLE
Majora Ansible Playbook v3.0.0

### PR DESCRIPTION
**Pull request by Oliboy50**
## Breaking changes

- Remove `hhvm` role (it didn't work, and `php7.x` is here to stay so I don't think we need this role anymore)
- Remove `ghostscript` role (it didn't work since Ubuntu 14.04, and it was only `apt-get install ...` so it can be replaced by the use of `extra_packages` variable for instance)
- Elastic products version (such as `elasticsearch_version`, etc.) no longer accept `'*.x'` values. The full version number must be provided now (also, default versions have been updated to `5.1.2`).

## Improvements

- Add `elasticsearch-cerebro` role (it's here to replace `elasticsearch-kopf` which doesn't support elasticsearch 5.x).
- Add `zsh_plugins` and `zsh_disable_auto_update` variables for `oh-my-zsh` role (also, `symfony2` plugin was added to `zsh_plugins` by default).
- All variables (except the "deprecated" ones) are now documented.
- Add some configuration variables for Elastic products.

## Bug fixes

- Fix `apollo` role
- Fix `elasticsearch` role (it was broken on some computers on `5.x` versions, because of the default JVM heap size of "2Go"... default value for the heap size is now "512Mo")
- Fix `elasticsearch-hq` role (it couldn't be provisioned twice+)
- Fix `kibana` role
- Fix `nginx` role (default `sites.[x].fastcgi_pass` configuration was broken if you didn't set `php_versions` variable)
- Fix `rabbitmq` role (it was broken if you didn't set `rabbitmq` variable)
- Fix `lua` role (typo with `luarocks_packages` variable)

### CODE REVIEW

File: logstash/tasks/main.yml [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+- name: Download logstash >= 5.0
+  get_url:
+    url: 'https://artifacts.elastic.co/downloads/logstash/logstash-{{ logstash_version }}.deb'
+    dest: '/tmp/logstash-{{ logstash_version }}.deb'
+  when: not (logstash_version|match('^[0-2]'))
 
-- name: Install logstash in /opt/logstash/bin
-  apt: name=logstash state=present
+- name: Install logstash
+  apt: deb='/tmp/logstash-{{ logstash_version }}.deb' state=present
```
Comment (jody-frankowski) [inline logstash/tasks/main.yml:16]: The apt module supports using an url in the deb field. See https://docs.ansible.com/ansible/apt_module.html

Comment (Oliboy50) [reply logstash/tasks/main.yml:16]: yes, I saw that too, but we're supporting Ansible version `1.9` as min version (see https://github.com/LinkValue/majora-ansible-playbook/blob/master/meta/main.yml)

so we cannot do that yet :'(

**Review by cmoysan — APPROVED** [review]
👍

**Comment by Oliboy50** [comment]
to `v3.0.0` (v2.x was not well tested enough 😢)

**Pull request merged**